docs/src/Atmos/Microphysics.md

```diff
@@ -0,0 +1,369 @@
+|``E_{col}``           | collision efficiency between rain drops and cloud droplets| -                        | ``0.8``               |
+|``C_{drag}``          | rain drop drag coefficient                                | -                        | ``0.55``              |
+|``a_{vent}, b_{vent}``| rain drop ventilation factor coefficients                 | -                        | ``1.5 \;``,``\; 0.53``|
+|``K_{therm}``         | thermal conductivity of air                               | ``\frac{J}{m \; s \; K}``| ``2.4 \cdot 10^{-2}`` |
+|``\nu_{air}``         | kinematic viscosity of air                                | ``\frac{m^2}{s}``        | ``1.6 \cdot 10^{-5}`` |
+|``D_{vapor}``         | diffusivity of water vapor                                | ``\frac{m^2}{s}``        | ``2.26 \cdot 10^{-5}``|
+
+## Rain drop size distribution
+
+The rain-drop size distribution is assumed to follow Marshall-Palmer distribution (Marshall Palmer 1948 eq. 1):
+```math
+\begin{equation}
+n(r) = n_{0_{MP}} exp\left(- \lambda_{MP} \; r \right)
+\end{equation}
+```
+where:
+ - ``r`` is the drop radius,
+ - ``n_{0_{MP}}`` and ``\lambda_{MP}`` are the Marshall-Palmer distribution parameters (twice the values used in the Marshall Palmer 1948, because we use drop radius and not diameter).
+
+## Terminal velocity
+
+The terminal velocity of an individual rain drop is defined by the balance between the gravity (taking into account the density difference between water and air) and the drag force:
+```math
+\begin{equation}
+v_{drop} = \left(\frac{8}{3 \; C_{drag}} \left( \frac{\rho_{water}}{\rho} -1 \right) \right)^{1/2} g^{1/2} \; r^{1/2} = v_c(\rho) \; g^{1/2} \; r^{1/2}
+\end{equation}
+```
+where:
+ - ``g`` is the gravitational acceleration,
+ - ``C_{drag}`` is the drag coefficient,
+ - ``\rho_{water}`` is the density of water,
+ - ``\rho`` is the density of air.
+
+The mass weighted terminal velocity ``v_t`` is defined following Ogura and Takahashi 1971
+```math
+\begin{equation}
+v_t = \frac{F_{rain}}{RWC}
+\end{equation}
+```
+where:
+ - ``F_{rain} = \int_0^\infty n(r) \; m(r) \; v_{drop}(r) dr`` is the vertical flux of rain drops,
+ - ``RWC = \int_0^\infty n(r) \; m(r) dr = \rho \; q_{rai}`` is the rain water content.
+
+Integrating over the assumed Marshall-Palmer distribution results in
+```math
+\begin{equation}
+RWC = \frac{8 \pi \; n_{0_{MP}} \; \rho_{water}}{\lambda_{MP}^4}
+\label{eq:lambda}
+\end{equation}
+```
+```math
+F_{rain} = \Gamma \left(\frac{9}{2} \right) \frac{4}{3} n_{0_{MP}} \; \pi \; \rho_{water} v_c(\rho) \; g^{1/2} (\lambda_{MP})^{-9/2}
+```
+Eliminating ``\lambda_{MP}`` between the two equations and dividing by ``RWC`` results in
+```math
+\begin{equation}
+v_t = \Gamma \left( \frac{9}{2} \right) \; \frac{v_c(\rho)}{6} \; (8 \pi)^{-1/8} \left( \frac{\rho}{\rho_{water}} \right)^{1/8} n_{0_{MP}}^{-1/8} g^{1/2} q_{rai}^{1/8} = v_{t_{c}}(\rho) \; n_{0_{MP}}^{-1/8} g^{1/2} q_{rai}^{1/8}
+\end{equation}
```

Comment:
I'd find it clearer to write this as 

v_t = \frac{4}{3}\pi \, \Gamma \left( \frac{9}{2} \right) v_c(\rho) \left( \frac{g}{\lambda_{MP}} \right)^{1/2} 

with \lambda_{MP} = \left( \frac{1}{8\pi n_{0_{MP}}} \frac{\rho}{\rho_{water}} q_{rai} \right)^{1/4}

This makes it clearer where length scales come from, and that you are using q_{rai} to get at \lambda. (Please check details.)

Reply:
I like it better too. I was trying to always keep the q_rai out in the open. But this way is clearer. I also changed the code implementation to reflect the documentation structure.